FTCBotCode/Competition24_25.java

```diff
@@ -36,15 +52,71 @@ public void runOpMode() {
       telemetry.addData("key", 321);
       telemetry.update();
       while (opModeIsActive()) {
-        // Put run blocks here.
+        //Drive
         horizontal = gamepad1.right_stick_x;
         vertical = gamepad1.right_stick_y;
         pivot = -gamepad1.left_stick_x;
-        frontRight.setPower(-pivot + (vertical - horizontal));
-        backRight.setPower(-pivot + vertical + horizontal);
-        frontLeft.setPower(pivot + vertical + horizontal);
-        backLeft.setPower(pivot + (vertical - horizontal));
-        telemetry.addData("key", 123);        
+        frontRight.setPower((-pivot + (vertical - horizontal)) * drivePower);
+        backRight.setPower((-pivot + (vertical + horizontal)) * drivePower);
+        frontLeft.setPower((pivot + (vertical + horizontal)) * drivePower);
+        backLeft.setPower((pivot + (vertical - horizontal)) * drivePower);
+        telemetry.addData("key", 123);
+
+
+        
+        //Arm
+        //Lift - Motor
+        if !gamepad1.left_bumper && !gamepad1.right_bumper {
+          list.setPower(0);
+        }
+        else if gamepad1.left_bumper {
+          //down
+          lift.setPower(-liftPower);
+        }
+        else if gamepad1.right_bumper {
+          //up
+          lift.setPower(liftPower);
+        }
+        else {
+          lift.setPower(0);
+        }
+
+        //Shoulder - Motor
+        shoulder.setPower((left_trigger - right_trigger) * shoulderPower);
```

Comment:
Isn't this supposed to be gamepad1.left_trigger and gamepad1.right_trigger

Reply:
I just fixed this, thank you!!!